pkg/reconciler/instances/ory/test/ory_integration_test.go

```diff
@@ -0,0 +1,63 @@
+package test
+
+import (
+	"context"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/kyma-incubator/reconciler/pkg/logger"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/kubernetes"
+	"github.com/kyma-incubator/reconciler/pkg/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubectl/pkg/util/podutils"
+)
+
+const (
+	envOryIntegrationTests = "ORY_RECONCILER_INTEGRATION_TESTS"
+	namespace              = "kyma-system"
+)
+
+func OryIntegrationTest(t *testing.T) {
```

Comment:
The name of the function does not reflect what it does. In fact it does not perform any test but checks for env to optionally skip the suite. What about changing the name to `skipTestIfDisabled()`?